jplag.endToEndTesting/src/main/java/de/jplag/endToEndTesting/helper/JsonHelper.java

```diff
@@ -0,0 +1,22 @@
+package de.jplag.endToEndTesting.helper;
+
+import java.nio.file.Paths;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import model.ResultJsonModel;
+
+public class JsonHelper {
```

Comment:
final class and private constructor (utility class)

Reply:
done